### DISCUSSION TITLE
Add Release Events to Audit logs

### DISCUSSION THREAD

**Discussion by ekcasey** [community/community#18756] [Code Security]
The [audit logs](https://docs.github.com/en/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/reviewing-the-audit-log-for-your-organization#audit-log-actions) don't seem to capture events related to releases.

As an organization owner I would like to audit release creation and updates. In particular, any updates to release assets seem critical to audit from the perspective of supply chain security.

**Comment by boylejj** [comment]
@ekcasey... Audit log Product Manager here. Can you help me understand what type of release events you would expect to see in the GitHub audit log?

**Reply by lontivero** [reply]
One of our releases was replaced a hour ago and there is not way to know  who nor how it happened. https://x.com/lontivero/status/1810835747324448989

**Comment by fproulx-boostsecurity** [comment]
@boylejj 100% still relevant. I'm doing supply chain incident response and the lack of audit log for this kind of modification is problematic.
